src/elementPlugins/shortText/src/ApplicationView.tsx

```diff
@@ -9,43 +9,70 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
   isEditable,
   isRequired,
   allResponses,
+  evaluator,
 }) => {
-  const [validationMessage, setValidationMessage] = useState('')
+  const [validationMessageDisplay, setValidationMessageDisplay] = useState('')
   const [value, setValue] = useState(initialValue?.text)
+  const [isValid, setIsValid] = useState<boolean>()
```

Comment:
I've just realised, for clarity sakes, we might want three state variables ('unknown', 'valid', 'invalid').

Not required change atm

Reply:
Yeah, I think `undefined`, `true`, `false` is working okay for now, but might run into issues down the line...